whitepapers/Tables.md

```diff
@@ -1,5 +1,114 @@
 # SQL Tables
 
+## Template
 
+### {ModelName}
+{Description}
 
+| Column | Type | Indexed | Nullable | FK | Default | Description |
+|:-------|:----:|:-------:|:--------:|:--:|:-------:|:------------|
+| | | ✓ | ✓ | ✓ | | |
 
+#### Other indices
+* `{column_name}`, `{column_name}`, ... [(unique)]
+* ...
+
+## Data
+
+### SourceUniqueIdentifier (SUID)
+Identifier for a specific document from a specific source.
+
+| Column | Type | Indexed | Nullable | FK | Default | Description |
+|:-------|:----:|:-------:|:--------:|:--:|:-------:|:------------|
+| `source_doc_id` | text |  |  |  |  | Identifier given to the document by the source |
+| `ingest_config_id` | int |  |  | ✓ |  | IngestConfig used to ingest the document |
```

Comment:
👍 `identifier`

I think linking suid to source makes sense if we only have one ingest config for that source enabled at a time. If we want to support multiple per source, though, we could harvest the same document in different formats (maybe from two complementary APIs). If we don't keep them separate, there could be a back-and-forth, deleting each other's states where the formats don't overlap.